capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -12,6 +12,32 @@ const DOWN_KEY = 'ArrowDown';
 const ENTER_KEY = 'Enter';
 let selectedSuggestionPosition = -1;
 
+/**
+ * Gets suggestions to display to the users
+ */
+function getSuggestionsForUserToChooseFrom() {
+  const val = SEARCH_INPUT.value.trim();
+  if (!val) {
+    return;
+  }
+  const fetchStr = '/search?query=' + val;
+  console.log('Fetch String: ', fetchStr);
+  fetch(fetchStr).then((response) => response.json()).then((suggestions) => {
+    getsAllProposedSuggestions(suggestions, val);
+  });
+}
+
+/**
+ * Sends final college list to the dashboard
+ */
+function sendCollegeInformationToDashboard() {
+  const listItems = document.querySelector('ul').children;
+  const listArray = Array.from(listItems);
+  const idArray = Array.from(listArray, (l) => l.id);
+  const urlHtml = 'dashboard.html?id=' + idArray.join();
```

Comment:
what does the `Html` in `urlHtml` refer to? Would something like `dashboardUrl` be more descriptive?

Reply:
great point! changed!